config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# The label used to mark namespaces to belong to an organization
+OrganizationLabel: appuio.io/organization
+# The fair use limit of memory usage per CPU core
+MemoryPerCoreLimit: 4Gi
+# Privileged* is a list of the given type allowed to bypass restrictions
+# Wildcards are supported (e.g. "system:serviceaccount:*" or "cluster-*-operator").
+PrivilegedGroups: []
+PrivilegedUsers: []
+PrivilegedRoles: []
+PrivilegedClusterRoles:
+  - cluster-admin
+# Key and values are anchored regexes
+AllowedNodeSelectors:
+  appuio.io/node-class: flex|plus
```

Comment:
Nitpick: The mix of regex and wildcard is IMO a bit confusing.

Reply:
The wildcards were picked for the additional flexibility of allowing `syn-resource-locker-*-manager` and similar. All the roles are ORed.

Here would be way more complicated definition/code IMO if we don't have regex support.